services/ses/index.js

```diff
@@ -17,6 +17,11 @@ function filterInternalErrors(error) {
   throw error;
 }
 
+function validateContact(contact) {
+  const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/; // eslint-disable-line max-len
```

Comment:
What do we use for the mailing list email input? I think they should be the same, it would be a bit strange if an email passed in one case but not the other.